ts/utils/__tests__/convertDateToWordDistance.tests.ts

```diff
@@ -0,0 +1,28 @@
+import { addHours, addYears, format } from "date-fns";
+import { convertDateToWordDistance } from "../convertDateToWordDistance";
+
+test("Compare now date with 2 hours earlier, expected  hh:mm", () => {
```

Comment:
we use spec like structure for tests, see [here](https://github.com/teamdigitale/italia-app/blob/7e6b792a1be4baa363e5eaaaf8d25913c8990794/ts/sagas/__tests__/messages.test.ts)